src/Type/Definition/Type.php

```diff
@@ -280,29 +283,14 @@ public static function isAbstractType($type) : bool
 
     /**
      * @param mixed $type
-     *
-     * @return mixed
      */
-    public static function assertType($type)
+    public static function assertType($type) : Type
```

Comment:
Personally I'd drop this method. Wonder what others think.

For me types eg. here are sufficient:

```php
    /**
     * @param callable():Type|Type $type
     */
    public function __construct($type)
    {
        $this->ofType = is_callable($type) ? $type : Type::assertType($type);
    }
```

Reply:
It seems to be a leftover from `graphql-js`, in JS you really can't ever be sure. Agreed that those assertions aren't really needed as PHP itself has runtime typechecks in function signatures.

Reply:
It's only being used in one place; I will nuke it in my next PR.